pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -71,6 +71,11 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) *apps.Deployment {
 	if c.HostNetwork {
 		podSpec.Spec.DNSPolicy = v1.DNSClusterFirstWithHostNet
 	}
+	prometheusAnnotations := map[string]string{
+		"prometheus.io/scrape": "true",
+		"prometheus.io/port":   strconv.Itoa(metricsPort),
+	}
+	podSpec.ObjectMeta.Annotations = prometheusAnnotations
```

Comment:
How about if you added these directly to the pod spec? For example, after the `Labels` on line 42 above. Other properties are just added after the initial spec definition if there is some conditional.

Reply:
@travisn thanks, your comment has been addressed